homework21/tests/test_library.py

```diff
@@ -0,0 +1,176 @@
+"""Тестирование библиотеки (pytest)"""
+
+import os
+from loguru import logger
+from ..source.Homework11_1 import Reader, Book
+
+
+def setup_logging(test_name, log_level="DEBUG"):
+    """
+    Logger setup.
+    """
+    logger.remove()
+    log_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                           "library_test_logs")
+    os.makedirs(log_dir, exist_ok=True)
+    log_file = os.path.join(log_dir, f"{test_name}.log")
+    logger.add(log_file, level=log_level, enqueue=True)
```

Comment:
You can use 1 logger across a single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application